spec/system/admin/settings_spec.rb

```diff
@@ -292,6 +292,21 @@
       expect(page).to have_current_path(admin_settings_path)
       expect(page).to have_css("div#tab-feature-flags.is-active")
     end
+
+    scenario "On #tab-sdg-configuration", :js do
+      sdg_setting = Setting.create!(key: "sdg.whatever")
+      admin = create(:administrator).user
+      login_as(admin)
+      visit admin_settings_path
+      find("#sdg-tab").click
```

Comment:
I know this is consistent with the rest of the file. However :sweat_smile:, since we're now trying to avoid this style as much as possible, how about:

```
    scenario "On #tab-sdg-configuration", :js do
      Setting.create!(key: "sdg.whatever")
      login_as(create(:administrator).user)

      visit admin_settings_path
      click_link "SDG configuration"

      accept_alert do
        within("tr", text: "Whatever") { click_button "Enable" }
      end

      expect(page).to have_current_path(admin_settings_path)
      expect(page).to have_css("h2", exact_text: "SDG configuration")
    end
```